_episodes/071-twir-442.md

```diff
@@ -0,0 +1,108 @@
+  Automotive](https://www.youtube.com/watch?v=MfG7OvfVYQY&t=2760s) by Christof
+  Petig/cpetig and Florian Gilcher/skade.)
+
+Allen focused predominantly on the web and developer productivity.
+
+- [Rocket web framework's v0.5 2nd release candidate being
+announced](https://rocket.rs/v0.5-rc/news/2022-05-09-version-0.5-rc.2/)
+- [RepliByte's release
+announcement](https://www.reddit.com/r/rust/comments/ukmnow/an_opensource_tool_to_seed_your_dev_database_with/)
+- [Rust Safety with Quentin Ochem and Florian Gilcher](https://rustacean-station.org/episode/067-quentin-ochem-florian-gilcher/)
+
+Sean decided to pick some fun geek topics.
+
+- [Kani Rust Verifier
+  Project](https://model-checking.github.io/kani-verifier-blog/2022/05/04/announcing-the-kani-rust-verifier-project.html)
+- [Over-Engineering A Fairly Simple
+Coding Challenge](https://ada-x64.github.io/over-engineering/)
+- [Parsing/Recursive Descent Parser](https://www.huy.rocks/everyday/05-08-2022-parsing-recursive-descent-parser)
+
+Tim's picks for the week were mostly related to security and enhancing the
+robustness of the crate ecosystem.
+
+- [Xilem](https://raphlinus.github.io/rust/gui/2022/05/07/ui-architecture.html), a
+UI architecture for Rust
+- [Securing
+Crates](https://tl8.co/entry/securing-crates) discussing side channel attacks
+- [Modeling Interconnected Social and Technical Risks in Open Source Software
+  Ecosystems](https://arxiv.org/abs/2205.04268), a related paper
+
+We hope that you enjoy the episode!
+
+### Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to
+suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps
+
+- [@00:12] Speaker introductions and agenda
+- [@01:20] Quote of the week
+- [@02:57] **Official updates**
+- [@03:01] Security advisory: `rustdecimal crate`
+- [@06:55] CTCFC Agenda
+- [@09:50] **Highlights from the newsletter**
+- [@10:15] Kani Rust Verifier project
+- [@20:29] Rocket web framework v0.5 2nd release candidate
+- [@23:35] Xilem
+- [@29:30] Over-engineering a fairly simple coding challenge
+- [@35:26] RepliByte's release announcement
+- [@39:07] Securing crates
+- [@44:09] Modeling Interconnected Social and Technical Risks in Open Source Software
+  Ecosystems
+- [@47:10] Parsing/Recursive Descent Parser
+- [@54:10] Rust Safety with Quentin Ochem and Florian Gilcher
+
+### Credits
+
+Intro Theme: [Aerocity](https://twitter.com/AerocityMusic)
+
+Audio Editing: [Aleksandar Nikolic](https://linkedin.com/in/dbsound)
+
+Hosting Infrastructure: [Jon Gjengset][jon]
+
+Show Notes: [Tim McNamara][tim]
+
+Hosts: [Tim McNamara][tim] and [Allen Wyma][allen]
```

Comment:
```suggestion
Hosts: [Tim McNamara][tim], [Allen Wyma][allen], and [Sean Chen][sean]
```

Reply:
Really sorry that I left you out Sean. I am sure that I had included you. It must have been on a previous draft.